File: 2309. Greatest English Letter in Upper and Lower Case/solution.cpp
class Solution {
public:
  string greatestLetter(string s) {
    unordered_set<char> mySet = unordered_set<char>(s.begin(),s.end());

    char lowercaseLetter = 'z';
    char uppercaseLetter = 'Z';

    for (int index = 0; index < 26; index++){
      if (mySet.count((char) ((int) uppercaseLetter - index)) == 1 && mySet.count((char) ((int) lowercaseLetter - index)) == 1){
        return string(1, (char) ((int) uppercaseLetter - index));
      }
    }

    return "";
  }
};

File: 2309. Greatest English Letter in Upper and Lower Case/solution.py
class Solution(object):
  def greatestLetter(self, s):
    """
    :type s: str
    :rtype: str
    """
    mySet = set(s)

    lowercaseLetter = ord('z')
    uppercaseLetter = ord('Z')

    for index in range(26):
			if chr(uppercaseLetter - index) in mySet and chr(lowercaseLetter - index) in mySet:
			  return chr(uppercaseLetter - index)
		
    return ''